scripts/get_release_branch.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Retrieve the branch name from the release PR"""
+import json
+import requests
+
+
+def check_for_release_pr(pull):
+    label = pull["head"]["label"]
+
+    if label.find("release/") != -1:
+        return pull["head"]["ref"]
+    else:
+        pass
```

Comment:
two nits:

* Can drop the `else` clause
* I think there's an implicit assumption being made here that there's only one release branch that ever exists at one time (I initially read this and wondered if we had to check that we're returning the right release branch). I think this is fine since I don't see how we'd end up in a situation where this isn't true, but it'd be good to call out in a comment